### ISSUE TITLE
fix - DemuxResults call in cg demultiplex report

### ISSUE THREAD

**Issue by diitaz93** [Clinical-Genomics/cg#2354]
## Description

When instantiating the class DemuxResults in `cg/cli/demultiplex/report.py:create_report_cmd`, only one out of 3 positional arguments are called, which raises an error when using the CLI command `cg demultiplex report <flow_cell_name>`.
```bash
[js.diazboada@hasta:/home/proj/stage/demultiplexed-runs] [S_base] 21s $ cg demultiplex report <some_flow_cell>
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
Running cg demultiplex.
Check demuxed flowcell <flow_cell>
Set environment to stage
Traceback (most recent call last):
  File "/home/proj/stage/bin/miniconda3/envs/S_main/bin/cg", line 8, in <module>
    sys.exit(base())
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/cg/cli/demultiplex/report.py", line 30, in create_report_cmd
    demux_dir=Path(demux_api.out_dir, flow_cell_name, flow_cell=flow_cell)
TypeError: __init__() missing 2 required positional arguments: 'flow_cell' and 'bcl_converter'
```

## Suggested solution

Add the missing arguments to the call of DemuxResults which are 'flow_cell' and 'bcl_converter'

## This can be closed when

The call of  `cg demultiplex report <flow_cell_name>` no longer raises an error.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by henrikstranneheim** [comment]
As discussed this will be fixed shortly when deprecating the Demultiplexing and cgstats repos. And the report is no longer relevant. Let's not spend time on this one now.
